bundles/com.espressif.idf.ui/OSGI-INF/l10n/bundle.properties

```diff
@@ -17,7 +17,7 @@ command.label.0 = Download and Configure ESP-IDF
 command.label.1 = ESP-IDF: Application Size Analysis
 command.name.3 = ESP-IDF: Application Size Analysis
 command.name.4 = Product Information
-command.label.4 = Product Information
+command.label.4 = List Product and IDF Tools Information
```

Comment:
To make it simple, we can still keep it as `Product Information` but while printing you can print existing product information first followed by tools information.

Reply:
Done